lib/fluent/plugin/in_unix.rb

```diff
@@ -80,22 +98,22 @@ def run
     # }
     def on_message(msg)
       # TODO format error
-      tag = msg[0].to_s
+      tag = @tag || (msg[0].to_s)
```

Comment:
should it check if tag is `nil`?

Reply:
Maybe, msg type check is needed. Will add it.